config/src/main/java/org/springframework/security/config/web/server/ServerHttpSecurity.java

```diff
@@ -2586,6 +2592,45 @@ public HeaderSpec referrerPolicy(Customizer<ReferrerPolicySpec> referrerPolicyCu
 			return this;
 		}
 
+		/**
+		 * Configures the <a href=
+		 * "https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cross-Origin-Opener-Policy">
+		 * Cross-Origin-Opener-Policy</a> header.
+		 * @return the {@link HeaderSpec} for additional customizations
+		 * @since 5.6
+		 * @see CrossOriginPoliciesServerHttpHeadersWriter
+		 */
+		public HeaderSpec crossOriginOpenerPolicy(CrossOriginOpenerPolicy openerPolicy) {
+			this.crossOriginPolicies.setOpenerPolicy(openerPolicy);
+			return this;
+		}
+
+		/**
+		 * Configures the <a href=
+		 * "https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cross-Origin-Embedder-Policy">
+		 * Cross-Origin-Embedder-Policy</a> header.
+		 * @return the {@link HeaderSpec} for additional customizations
+		 * @since 5.6
+		 * @see CrossOriginPoliciesServerHttpHeadersWriter
+		 */
+		public HeaderSpec crossOriginEmbedderPolicy(CrossOriginEmbedderPolicy embedderPolicy) {
+			this.crossOriginPolicies.setEmbedderPolicy(embedderPolicy);
+			return this;
+		}
+
+		/**
+		 * Configures the <a href=
+		 * "https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cross-Origin-Resource-Policy">
+		 * Cross-Origin-Resource-Policy</a> header.
+		 * @return the {@link HeaderSpec} for additional customizations
+		 * @since 5.6
+		 * @see CrossOriginPoliciesServerHttpHeadersWriter
+		 */
+		public HeaderSpec crossOriginResourcePolicy(CrossOriginResourcePolicy resourcePolicy) {
```

Comment:
The same feedback that I mentioned in `HeadersConfigurer.java` applies here. For consistency, we should stick to the lambda DSL.